src/main/scala/com/dwolla/cloudflare/DnsRecordClient.scala

```diff
@@ -38,9 +38,9 @@ class DnsRecordClient(executor: CloudflareApiExecutor)(implicit val ec: Executio
     }.map((_, record.zoneId))
   }
 
-  def getExistingDnsRecord(name: String, content: Option[String]=None, recordType: Option[String]=None): Future[Option[IdentifiedDnsRecord]] = {
+  def getExistingDnsRecord(name: String, contentPredicate: String ⇒ Boolean, recordType: Option[String] = None): Future[Option[IdentifiedDnsRecord]] = {
```

Comment:
I'm concerned about this being a breaking change from the version we released yesterday, but I don't want to bump the version to 2.0 for this change. Can this be implemented alongside the old method?

Reply:
Updated. Note I couldn't do a straight overload of the existing `getExistingDnsRecord` method due to a [limitation](https://groups.google.com/forum/#!msg/scala-user/FyQK3-cqfaY/fXLHr8QsW_0J) of Scala.